lib/neo4j/active_node/has_n.rb

```diff
@@ -463,13 +463,14 @@ def define_has_one_id_methods(name)
         end
       end
 
-      def define_has_one_getter(name)
+      def define_has_one_getter(name) # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Rubocop was failing this with PerceivedComplexity as too high. I didn't know what to do with that, so I suppressed the warning to make it pass 😕 ...

Reply:
It could probably just do with an extraction of a method.  I'll fix it and link to the commit here

Reply:
Meh, not the best: https://github.com/neo4jrb/neo4j/commit/1122ee7b302b1d1d7b29b56d313987ac95913abe